posdealers/04-after-sales/how-to-switch-scu.md

```diff
@@ -5,4 +5,61 @@ title: 'How-To: SCU austauchen'
 
 # How-To: SCU austauchen
 
-Wir werden in Kürze eine Anleitung zur Verfügung stellen.
+Diese Anleitung beschreibt, wie Sie in einer Cashbox die Zuordnung einer Queue zur verwendeten SCU mit einer neuen SCU austauschen. Das hier beschriebene Feature ist hilfreich für den Fall, dass Sie eine TSE austauschen möchten.
+
+## Schritte
+
+Generell sind zwei Fälle für den Austausch zu unterscheiden:
+
+1. Die Queue ist noch nicht aktiv (Inbetriebnahme-Beleg wurde noch nicht gesendet)
+2. Die Queue ist bereits aktiv (Inbetriebnahme-Beleg wurde bereits gesendet)
+
+## Queue ist noch nicht aktiv
+
+### SCU befindet sich in derselben Cashbox wie die Queue
+
+Im diesem Fall ist der Austauch unkompliziert und besteht aus folgenden Schritten:
+
+1. Alte SCU aus der Cashbox entfernen
+2. Neue SCU in die Cashbox einfügen
+3. Queue mit neuer SCU verbinden
+4. Konfigurationsänderung publizieren (rebuild configuration)
+5. fiskaltrust Service neu starten
```

Comment:
Sollen wir hier auch beim "fiskaltrust Middleware"-Wording bleiben?

Reply:
fixed in https://github.com/fiskaltrust/getting-started/commit/1842b15d660cf54d58c0e53df4ffb2ea12f2d0c6